app/Support/PluginManager.php

```
@@ -1885,6 +2068,249 @@ public function loadActivePlugins(): void
+            'plugin_id' => $pluginId,
+            'restored' => $restored,
+            'error' => $cause->getMessage(),
+        ]);
+    }
+
+    /** @param list<array<string,mixed>> $hooks */
+    private function restorePluginHooks(int $pluginId, array $hooks): void
+    {
+        $delete = $this->db->prepare('DELETE FROM plugin_hooks WHERE plugin_id = ?');
+        if ($delete === false) {
+            throw new \RuntimeException('Impossibile ripristinare gli hook del plugin.');
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '2268', column '23'). (undefined)

(MissingImport)